Prog/clean-arlington.Rmd

```diff
@@ -1052,9 +1053,121 @@ rm(mult, other, nunits, cdties)
 
 ```
 
+
+## Add additional variables 
+
+Get additional metric for number of units from the Arlington County address file, which was downloaded in the preclean step.
+
+```{r address-file}
+
+# read address file
+rdir <- paste0("L:/Libraries/RegHsg/Raw/", filepath, "/")
+adfile <- paste0(rdir, filepath, "-address-file.csv")
+address <- read_csv(adfile)
+
+address1 <- address %>% 
+  mutate(address_cap = toupper(FULL_ADDRESS)) %>% 
+  select(address_cap, units_address = UNITCOUNT)
+
+jur5 <- jur4 %>% 
+  mutate(address_cap = toupper(propaddress)) %>% 
+  left_join(address1, by = "address_cap") %>% 
+  select(-address_cap)
+
+
+```
+
+Use a spatial join to the parcel shapefule to fill in lot size.
+
+```{r spatial}
+
+par <- st_read(dsn = "L:/Libraries/RegHsg/Maps/arlington-parcel-map",
+               layer = "Parcel_Polygons") %>% 
+  st_transform(crs = 4326)
+
+# calculate area in square feet
+par <- par %>% 
+  mutate(area = st_area(par) %>% units::set_units("feet^2") %>% units::set_units(NULL))
+
+# convert arlington to spatial
+jursf <- jur5 %>% 
+  filter(!is.na(lat), !is.na(long)) %>% 
+  mutate(longx = long, laty = lat) %>% 
+  st_as_sf(coords = c("longx", "laty")) %>% 
+  st_set_crs(st_crs(4326))
+
+# join to zoning layer
+jurj <- st_join(jursf, par, st_intersects)
```

Comment:
do i need to be concerned about this one? 

"although coordinates are longitude/latitude, st_intersects assumes that they are planar"

Reply:
@lhendey no :)